R/demographics-table.R

```diff
@@ -0,0 +1,80 @@
+
+
+smr <- function(value = seq(1,5)) {
+  tibble(
+    `mean (sd)` = paste0(sig(mean(value, na.rm = TRUE)), " (", sig(sd(value,na.rm=TRUE)), ")"),
+    `min-max` = paste0(sig(range(value,na.rm=TRUE)), collapse = " - ")
+  )
+}
+
+#' This is the title
+#'
+#' This function makes a single table from both continuous and categorical data.
+#'
+#' @param data
+#' @param cols_cont
+#' @param cols_cat
+#'
+#' @return
+#' An object of class `pmtable`
+#'
+#' @examples
+#'
+#' @seealso [pt_cont_long()], [pt_cat_long()]
+#'
+#' @export
+pt_demographics <- function(data, cols_cont, cols_cat, span, units = NULL,
+                            stat_name = "Statistic", stat_width = 2) {
+
+  assert_that(is.data.frame(data))
```

Comment:
Let's check that `cols_cont` and `cols_cat` exist in the data frame.

Reply:
Look in `check.R` ... there are functions for checking discrete columns and continuous columns.